server/chat-commands/room-settings.ts

```diff
@@ -15,6 +15,26 @@ const SLOWCHAT_MINIMUM = 2;
 const SLOWCHAT_MAXIMUM = 60;
 const SLOWCHAT_USER_REQUIREMENT = 10;
 
+export const sections = [
+	'official', 'smogontiers', 'communityprojectsformats', 'languages', 'entertainment', 'gaming', 'lifehobbies', 'onsitegames',
+] as const;
+
+export type RoomSection = typeof sections[number];
+
+export const RoomSections: {sectionNames: {[k in RoomSection]: string}, sections: readonly RoomSection[]} = {
+	sectionNames: {
+		official: 'Official',
+		smogontiers: 'Smogon tiers',
```

Comment:
```suggestion
		smogonformats: 'Smogon formats',
```

To match "Community formats", although apparently we're considering merging them in the UI.